service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -50,16 +57,33 @@ public void handleDocumentUploadedEvent(final FurtherEvidenceUploadedEvent event
         recipients.removeIf(email -> Objects.equals(email, uploader.getEmail()));
 
         if (isNotEmpty(recipients)) {
-            furtherEvidenceNotificationService.sendNotification(caseData, recipients, uploader.getFullName());
+            furtherEvidenceNotificationService.sendNotification(caseData, recipients, uploader.getFullName(),
+                newNonConfidentialDocuments);
         }
     }
 
-    private static boolean hasNewNonConfidentialDocuments(List<Element<SupportingEvidenceBundle>> newEvidenceBundle,
-                                                          List<Element<SupportingEvidenceBundle>> oldEvidenceBundle) {
-        List<SupportingEvidenceBundle> oldEvidenceBundleUnwrapped = unwrapElements(oldEvidenceBundle);
+    private List<String> getNewNonConfidentialDocuments(List<Element<SupportingEvidenceBundle>> newEvidenceBundle,
```

Comment:
would rename this to `getNewNonConfidentialDocumentsNames` or something like that, we are not getting the actual docs